js/index.js

```diff
@@ -1,7 +1,48 @@
 // Iteration 1: Names and Input
-
+let hacker1 = "Elisenda"
+console.log(`The drivers name is ${hacker1}`)
+let hacker2 = "Felipe"
+console.log(`The navigator name is ${hacker2}`)
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+  console.log(`The driver has the longest name, it has ${hacker1.length} characters`)
+} else if (hacker1.length < hacker2.length) {
+hacker2.length} 
+else if (hacker1.length == hacker2.length) {
+  console.log("Wow, you both have equally long names, characters!")
+} else {
+  console.log("Woow, you're both a hackers")
+}
+
+
+
+// Iteration 3: Loops
+
+//3.1
+let text = "";
+let person = hacker2.toUpperCase().split("").join(" ")
+let dirver;
+for (dirver in person) {
+  text += person[dirver] + " ";
+}
```

Comment:
bien, funciona, pero mejor usar un for loop clásico para iterar sobre strings, el objetivo del for...in loop es de iterar sobre propiedades de objeto.